examples/incremental_learning/helmet-detection-modelbox/incremental_learning-modelbox/src/flowunit/yolo3_post/yolo3_utils.py

```diff
@@ -0,0 +1,273 @@
+
+# from sedna.common.config import Context
+# from sedna.common.file_ops import FileOps
+# from sedna.core.incremental_learning import IncrementalLearning
+# from interface import Estimator
+
+# he_saved_url = Context.get_parameters("HE_SAVED_URL", '/tmp')
+# rsl_saved_url = Context.get_parameters("RESULT_SAVED_URL", '/tmp')
+
+# class_names = ['person', 'helmet', 'helmet_on', 'helmet_off']
+
+# car, bus, truck
+coco_car_classes = [2, 5, 7]
+
+def draw_boxes(img, labels, scores, bboxes, class_names, colors):
+    line_type = 2
+    text_thickness = 1
+    box_thickness = 1
+    #  get color code
+    colors = colors.split(",")
+    colors_code = []
+    for color in colors:
+        if color == 'green':
+            colors_code.append((0, 255, 0))
+        elif color == 'blue':
+            colors_code.append((255, 0, 0))
+        elif color == 'yellow':
+            colors_code.append((0, 255, 255))
+        else:
+            colors_code.append((0, 0, 255))
+    label_dict = {i: label for i, label in enumerate(class_names)}
+    for i in range(bboxes.shape[0]):
+        bbox = bboxes[i]
+        if float("inf") in bbox or float("-inf") in bbox:
+            continue
+        label = int(labels[i])
+        score = "%.2f" % round(scores[i], 2)
+        text = label_dict.get(label) + ":" + score
+        p1 = (int(bbox[0]), int(bbox[1]))
+        p2 = (int(bbox[2]), int(bbox[3]))
+        if (p2[0] - p1[0] < 1) or (p2[1] - p1[1] < 1):
+            continue
+        # try:
+            cv2.rectangle(img, p1[::-1], p2[::-1],
+                          colors_code[labels[i]], box_thickness)
+            cv2.putText(img, text, (p1[1], p1[0] + 20 * (label + 1)),
+                        cv2.FONT_HERSHEY_SIMPLEX, 0.6, (255, 0, 0),
+                        text_thickness, line_type)
+        # except TypeError as err:
+        #     warnings.warn(f"Draw box fail: {err}")
+    return img
+
+def output_deal(infer_result, nframe, img_rgb, class_names):
+# def output_deal(is_hard_example, infer_result, nframe, img_rgb):
+    # save and show image
+    img_rgb = np.array(img_rgb)
+    img_rgb = cv2.cvtColor(img_rgb, cv2.COLOR_RGB2BGR)
+    colors = 'yellow,blue,green,red'
+
+    lables, scores, bbox_list_pred = infer_result
+    img = draw_boxes(img_rgb, lables, scores, bbox_list_pred, class_names,
+                     colors)
+    # if is_hard_example:
+    #     cv2.imwrite(f"{he_saved_url}/{nframe}.jpeg", img)
+    # cv2.imwrite(f"{rsl_saved_url}/{nframe}.jpeg", img)
+
+
+def mkdir(path):
+    path = path.strip()
+    path = path.rstrip()
+    is_exists = os.path.exists(path)
+    if not is_exists:
+        os.makedirs(path)
+
+
+def post_process(model_output):
+    all_classes, all_scores, all_bboxes = model_output
+    rsl = []
+    for c, s, bbox in zip(all_classes, all_scores, all_bboxes):
+        bbox[0], bbox[1], bbox[2], bbox[3] = bbox[1], bbox[0], bbox[3], bbox[2]
+        rsl.append(bbox.tolist() + [s, c])
+    return rsl
+
+
+
+
+
```

Comment:
So many empty lines is not suggestted, please follow the standard code style, for example google python code style, or you can use the IDE to format the code.